testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/EnvSinglenodeDeltaLakeOss.java

```diff
@@ -124,7 +124,7 @@ public void extendEnvironment(Environment.Builder builder)
     @SuppressWarnings("resource")
     private DockerContainer createSparkContainer()
     {
-        DockerContainer container = new DockerContainer("ghcr.io/trinodb/testing/spark3-delta:" + hadoopImagesVersion, SPARK_CONTAINER_NAME)
+        DockerContainer container = new DockerContainer("ghcr.io/trinodb/testing/spark4-delta:" + hadoopImagesVersion, SPARK_CONTAINER_NAME)
```

Comment:
Could you also update `SparkDeltaLake` class?

https://github.com/trinodb/trino/blob/39657bdffe31bf689fafca4c093305d5055d3120/plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/SparkDeltaLake.java#L36